Geometry/Records/interface/CaloTowerGeometryRecord.h

```diff
@@ -27,6 +28,7 @@ class CaloTowerGeometryRecord :
                 IdealGeometryRecord,
 		CaloTowerAlignmentRcd, 
 		CaloTowerAlignmentErrorRcd,
```

Comment:
@ianna to me, *AlignmentErrorRcd should stop existing after everything is tested and validated. And in principle the extended format should become main. 

Otherwise, if you insists I can move the *AlignmentErrorRcd includes to the corresponding*AlignementErrorExtendedRcd.